releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,24 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 7.0.0 - November 21, 2023
+### Breaking Changes
+* Removed .NET Core 3.1 as a target framework as .NET Core 3.1 is out of support. Updated `Microsoft.Extensions` dependencies to version 6.0.0 and later. [#482](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/482)
+* `AddAzureAppConfiguration` now throws errors for invalid inputs when the `optional` flag is equal to `true`. [#408](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/408)
+
+### Enhancements
+* This is the first stable release of the following API introduced in the 7.0.0-preview release. [#422](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/422)
```

Comment:
Please link to the issue instead of the PR whenever possible. The issue provides more context (and contains the link to the PR).